config.json

```diff
@@ -676,7 +676,13 @@
         "name": "Word Count",
         "uuid": "0073ff9a-cd6a-43cf-b8bf-4d5d8117b81b",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "for-loops",
+          "regular-expressions",
+          "arrays",
+          "objects"
+        ],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/word-count/solutions/TomPradat

I chose `for-loops` instead of `array-transformations` given the difficulty of the exercise. I omitted `conditionals` as the concept is a prerequisite of `objects` and `for-loops`

Reply:
By the way, I think the difficulty of this exercise is underrated, regex are not that easy to deal with, I'd give a 4 or 5. If you agree I can make a new PR ? Or just a separated commit on this PR

Reply:
Make a new PR but yeah let's bump it.